### PR TITLE
ps  removed createdBy in controller and added a name to all the messages

### PR DESCRIPTION
**Pull request by SeanRichhh**

### CODE REVIEW

File: src/domain/conversation.js [hunk 1]
```diff
@@ -64,6 +64,26 @@ export default class Conversation {
...
+            conversationId: conversation.id
+          },
+          include: {
+            user: { include: { profile: true } }
+          }
+        })
+        newMessages = newMessages.map((newMessage) => {
+          const createdBy =
+            newMessage.user.profile.firstName +
+            ' ' +
```
Comment (Auenc) [inline src/domain/conversation.js:80]: Can we change this to string interpolation please?

**Review by Auenc — CHANGES_REQUESTED** [review]
Just a small change to how we are creating the `createdBy` value

File: src/domain/conversation.js (hunk 1 above)
Comment (SeanRichhh) [reply src/domain/conversation.js]: Just changed it

**Review by shahzadnaeem — APPROVED** [review]
Lewis says yes!

**Pull request merged**